app/bt/AffectedUserFlow/CodeInput.tsx

```diff
@@ -131,6 +164,21 @@ const CodeInputScreen = (): JSX.Element => {
   );
 };
 
+const LoadingIndicator = () => {
```

Comment:
just a question, not sure what the answer is, but should we have 1 or 2 loading indicators that we store in a single place and use throughout the app for stylistic consistency? 🤔

Reply:
@devinjameson and I chatted about this. We arent 100% sure at this point, as this is the only place we want this right now, but if we do have a few it makes sense to abstract at least the styles out to be reused. Either way it would makes sense in a follow up PR and we cut a card for this work here: https://trello.com/c/bBehk7Pr/168-as-a-user-i-see-a-consistent-and-well-styled-loading-indicator-throughout-app-when-data-is-being-loading